src/platforms/react-native/upload-debug.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Debug Symbols
+sidebar_order: 96
+description: "Learn about using the Sentry Gradle Plugin or sentry-cli for providing debug symbols to Sentry."
+---
+
+Debug symbols provide information that Sentry displays on the **Issue Details** page to help you triage an issue. We offer a range of methods to provide Sentry with debug symbols.
+
+## When to Upload
+
+With default settings, complete stack traces are available in your Java/Kotlin error, out of the box, unless you use [enableProguardInReleaseBuilds](https://reactnative.dev/docs/signed-apk-android#enabling-proguard-to-reduce-the-size-of-the-apk-optional). In this case, you need to upload the ProGuard/R8 mapping files generated by the [Android Gradle Plugin](https://developer.android.com/studio/build/shrink-code#shrink-code), so Sentry can show you proper stack traces.
+
+Errors raised from the native layer in Android apps require certain debug information files to be uploaded. For example, an Android app with [NDK](https://developer.android.com/ndk/) support, dwarf debug files need to be uploaded.
```

Comment:
```suggestion
Errors raised from the native layer in Android apps require certain debug information files to be uploaded. For example, for an Android app with [NDK](https://developer.android.com/ndk/) support, dwarf debug files need to be uploaded.
```
Is here `for` missing or am I not getting the sentence?

Reply:
I think its ambigual since the sentence starts with `For example`, then, `an Android app...`

Reply:
I've updated the suggestion, maybe it's better like that?

Reply:
This is one of the few cases where passive voice might be a good idea: 
```suggestion
Errors raised from the native layer in Android apps require certain debug information files to be uploaded. For example, dwarf debug files need to be uploaded for an Android app with [NDK](https://developer.android.com/ndk/) support.
```